tests/FSharp.TestHelpers/CompilerAssert.fs

```diff
@@ -1,7 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
 
-[<AutoOpen>]
-module FSharp.Compiler.UnitTests.CompilerAssert
+namespace FSharp.TestHelpers
```

Comment:
Is there a reason to move it from FSharp.Compiler to FSharp namespace?

Reply:
It'll probably change again, TestHelpers is pretty useless too.  It's likely to encapsulate general Testing infrastructure, Interactive infra as well as compiler.  The primary purpose of this pass is to get it separate from FSharp.TestSuite so that we can start teasing things apart and re-categorize tests so that they don't represent a 12 year old organization structure.

Reply:
I don't mind `CompilerAssert` -> `TestHelpers` change, my concern is only about `FSharp.Compiler` -> `FSharp` parent namespace.

> It's likely to encapsulate general Testing infrastructure, Interactive infra as well as compiler.

Interactive infra lives in `FSharp.Compiler.Interactive`, so there's probably no reason to move tests infra to a namespace not starting with `FSharp.Compiler`.

Reply:
@auduchinok , @vzarytovskii is a new full time member of the FSharp team, he is tasked with fixing up our testing.  He will undoubtedly be making proposals for testing tools namespaces, apis, categorizations, factorization and the like.  

As for the namespace it isn't going to live in the FSharp namespace ... but it probably won't be FSharp.Compiler either.  FSharp.Test, FSharp.Testing or FSharp.UnitTesting would be likely candidates, although others spring to mind too.  But right now he is just getting the feel of the problem to be solved, and our hideous test codebase.

Right now Vlad, is figuring out how the code goes together, and along the way moving things around to make things more convenient.

BTW.  did you ever get around to the test cases for this PR: https://github.com/dotnet/fsharp/pull/9241.

Reply:
> @vzarytovskii is a new full time member of the FSharp team

Hi Vlad, nice to meet you!

> BTW. did you ever get around to the test cases for this PR: #9241.

Nope, I've just returned to the computer after few days off the computer and still catching up in Mail/Slack/etc. :) I'll get there soon.

Reply:
@auduchinok I guess, my idea was that it contains not only Compiler-related helpers, but Test-related in general (like TestFramework, and Utilities).
As Kevin said, it will eventually move to a dedicated testing NS, likely along with other test utilities. Right now I'm trying to come up with proposal for how to rework testing story for this repo.

Reply:
Where can I find the list of F# team members? As far as I know there are
@dsyme 
@cartermp 
@KevinRansom 
@vzarytovskii 
Anyone I missed?

Reply:
@brettfo, @nelson-wu although nelson is still at school, and comes back later in the year.

Reply:
> Anyone I missed?

At least @TIHan and @brettfo who does many things too. 🙂

Reply:
Then there are the dotnet-interactive guys, who are also part of the team, @colombod , @jonsequitur  and maria, who's github ID I don't know.

Reply:
@auduchinok , time off is good.